MeDrop/MeDrop/Sources/Collection/View/CollectionView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CollectionView.swift
+//  MeDrop
+//
+//  Created by JAESEOK LEE on 2023/07/10.
+//
+
+import Foundation
+import SwiftUI
+
+struct CollectionView: View {
+    @State var sortedBy = "가나다 순"
+    
+    let screenWidth: Double
+    let cardWidth: Double
+    let cardHeight: Double
+    let cardRatio: Double
+    
+    init() {
+        self.screenWidth = UIScreen.main.bounds.size.width
+        cardWidth = screenWidth - .spacing40
+        cardHeight = cardWidth * widthHeightRatio
+        cardRatio = cardWidth / 348
+    }
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            VStack {
+            }
+            .frame(maxWidth: .infinity, maxHeight: .spacing32 * cardRatio, alignment: .trailing)
+            
+            Text("Collection")
+                .font(.black(34 * cardRatio))
+                .frame(maxWidth: .infinity, alignment: .leading)
+            
+            Menu {
+                sortingButton(order: "가나다 순")
+                sortingButton(order: "새로운 순")
+                sortingButton(order: "오래된 순")
```

Comment:
```suggestion
                ForEach(["가나다 순", "새로운 순", "오래된 순"], id: \.self) {
                    sortingButton(order: $0)
                }
```

ForEach를 사용하면 유지보수에 도움이 될 거같아요 !

Reply:
감사합니다. 반영하겠습니다.